src/service/notify.ts

```diff
@@ -5,7 +5,7 @@ import * as config from '../config';
 
 export const confirmationEmail = async (emailAddress: string, reference_number: string): Promise<void> => {
```

Comment:
To reduce complexity and remove duplication I would have added getUserEmail method here and remove the need to add logic on the check-your-answer page and on the getUserEmail util, something like:
```suggestion
export const confirmationEmail = async (jwt: string, reference_number: string): Promise<void> => {
    if (isFeatureEnabled(config.FEATURE_FLAG_ENABLE_NOTIFY)) {
        const notifyClient = new NotifyClient(config.NOTIFY_API_KEY);
        const emailAddress = getUserEmail(jwt);

        log.info(`Email confirmation is enabled. User: ${emailAddress}.`);

        await notifyClient.sendEmail(config.NOTIFY_EMAIL_TEMPLATE, emailAddress, {
           ...
        });
    } else {
        log.info('Email confirmation is disabled.');
    }
};
```